CMakeLists.txt

```diff
@@ -302,6 +302,8 @@ if (LLAMA_CUBLAS)
             set(LLAMA_EXTRA_LIBS ${LLAMA_EXTRA_LIBS} CUDA::cudart CUDA::cublas CUDA::cublasLt)
         endif()
 
+        set(LLAMA_EXTRA_LIBS ${LLAMA_EXTRA_LIBS} CUDA::cuda_driver)
+
```

Comment:
By the way `LLAMA_EXTRA_LIBS` ends up in `target_link_libraries(... PUBLIC ...)` which probably isn't what was intended here?